openedx/features/edly/api/v1/urls.py

```diff
@@ -3,6 +3,6 @@
 from openedx.features.edly.api.v1.views.user_sites import UserSitesViewSet
 
 router = routers.SimpleRouter()
-router.register(r'user-sites', UserSitesViewSet, base_name='user-sites')
+router.register(r'user_sites', UserSitesViewSet, base_name='user_sites')
```

Comment:
why this change?

Reply:
That was also a requirement of the mobile team.

Reply:
oh okay